python_modules/dagster/dagster/_core/storage/dagster_run.py

```diff
@@ -329,6 +258,29 @@ class DagsterRun(
         partitions_subset (Optional[PartitionsSubset]): The subset of partitions to execute.
     """
 
+    job_name: PublicAttr[str]
+    run_id: PublicAttr[str]
+    run_config: PublicAttr[Mapping[str, object]]
+    asset_selection: Optional[AbstractSet[AssetKey]]
+    asset_check_selection: Optional[AbstractSet[AssetCheckKey]]
+    op_selection: Optional[Sequence[str]]
+    resolved_op_selection: Optional[AbstractSet[str]]
+    step_keys_to_execute: Optional[Sequence[str]]
+    status: PublicAttr[DagsterRunStatus]
+    tags: PublicAttr[Mapping[str, str]]
+    root_run_id: Optional[str]
+    parent_run_id: Optional[str]
+    job_snapshot_id: Optional[str]
+    execution_plan_snapshot_id: Optional[str]
+    remote_job_origin: Optional[RemoteJobOrigin]
+    job_code_origin: Optional[JobPythonOrigin]
+    has_repository_load_data: bool
+    run_op_concurrency: Optional[RunOpConcurrency]
+
+    # Only support storing time window partitions subsets on the run for now, other
```

Comment:
oops this comment is not accurate in master either, maybe should just be 'certain subsets'

in hindsight this was probably not the smartest choice for forwards compatibility if we ever start sticking default parittions subsets in there in certain cases, ah well